src/runtime/components/data/Table.vue

```diff
@@ -152,14 +152,16 @@ export default defineComponent({
       default: undefined
     }
   },
-  emits: ['update:modelValue'],
+  emits: ['update:modelValue', 'update:sort'],
   setup (props, { emit, attrs: $attrs }) {
     const { ui, attrs } = useUI('table', toRef(props, 'ui'), config, toRef(props, 'class'))
 
     const columns = computed(() => props.columns ?? Object.keys(omit(props.rows[0] ?? {}, ['click'])).map((key) => ({ key, label: upperFirst(key), sortable: false })))
 
     const sort = ref(defu({}, props.sort, { column: null, direction: 'asc' }))
 
+    const defaultSort = { column: sort.value.column, direction: null }
```

Comment:
Why not store the default direction here?

```suggestion
    const defaultSort = { column: sort.value.column, direction: sort.value.direction }
```

Reply:
I honestly don't remember, but anyway it gets overridden by defu so there should be no side effects.